### PR TITLE
Renamed config vars

### PR DESCRIPTION
**Pull request by pallavi2209**
Renamed following config vars:
- DISABLE_HTTP --> REQUIRE_HTTPS
- USE_ACCESS_TOKEN --> REQUIRE_ACCESS_TOKEN
- HEROKU_CLOCK_DYNO --> ENABLE_CLOCK_PROCESS
- USE_WORKER_PROCESS --> ENABLE_WORKER_PROCESS
- payloadLimit --> REQUEST_PAYLOAD_LIMIT
- RATE_LIMIT --> DDOS_RATE_LIMIT
- RATE_WINDOW --> DDOS_RATE_WINDOW
- ENDPOINT_TO_LIMIT --> DDOS_ENDPOINT_TO_LIMIT
- HTTP_METHOD_TO_LIMIT --> DDOS_HTTP_METHOD_TO_LIMIT

Added following config vars in Sandbox :
ENABLE_CLOCK_PROCESS true
ENABLE_WORKER_PROCESS true

Added following config vars in Staging :
ENABLE_CLOCK_PROCESS true
ENABLE_WORKER_PROCESS true
REQUIRE_ACCESS_TOKEN false

Added following config vars in Production :
REQUIRE_HTTPS true
REQUIRE_ACCESS_TOKEN true
ENABLE_CLOCK_PROCESS true
ENABLE_WORKER_PROCESS true
DDOS_RATE_LIMIT 2000
DDOS_RATE_WINDOW 10000
DDOS_ENDPOINT_TO_LIMIT *
DDOS_HTTP_METHOD_TO_LIMIT post

### CODE REVIEW

File: config/toggles.js [hunk 1]
```diff
@@ -51,19 +51,19 @@ function environmentVariableTrue(processEnv, environmentVariableName) {
 const longTermToggles = {
 
   // Disable HTTP, i.e. only use https
-  disableHttp: environmentVariableTrue(pe, 'DISABLE_HTTP'),
+  disableHttp: environmentVariableTrue(pe, 'REQUIRE_HTTPS'),
```
Comment (iamigo) [inline config/toggles.js]: I know this isn't specifically in the description (sorry), but I think it'll be easier for everyone in the long run if the toggle attribute names are renamed to requireHttps, enableClockProcess, requireAccessToken, enableWorkerProcess.

(And keep them in alphabetical order here, which will make it easier when there are more of them.)

File: docs/docs/01-quickstart.md [hunk 2]
```diff
@@ -296,7 +296,7 @@ POST to `v1/subjects/USA/child` using the following body:
 ```
 ### Request Rate Limiting
 
-Refocus lets you limit requests based on IP address using environment variables. Set `RATE_WINDOW` to the time interval in milliseconds after which the limits are reset and set `RATE_LIMIT` to the total number of requests allowed per IP. When the request limit is exceeded, the request is rejected with HTTP status code 429.
-For example, if you set `RATE_WINDOW=300000` and `RATE_LIMIT=250`, then only 250 requests will be allowed from any individual IP address over any rolling five minute period.
+Refocus lets you limit requests based on IP address using environment variables. Set `DDOS_RATE_WINDOW` to the time interval in milliseconds after which the limits are reset and set `DDOS_RATE_LIMIT` to the total number of requests allowed per IP. When the request limit is exceeded, the request is rejected with HTTP status code 429.
```
Comment (iamigo) [inline docs/docs/01-quickstart.md]: Thx for updating the docs, too!

**Comment by pallavi2209** [comment]
@iamigo @annyhe updated the PR
I was thinking if there could be a better way to refer toggle names. Like if we have to change sampleStore toggle name, we will have to change it in > 10 files. I thought of creating a constant, but then same logic applies to constant name as well.

File: config.js [hunk 3]
```diff
@@ -41,10 +41,10 @@ const DEFAULT_CHECK_TIMEOUT_INTERVAL_MILLIS = 30000;
 const CACHE_EXPIRY_IN_SECS = 60;
 
 // request limiter settings
-const rateLimit = pe.RATE_LIMIT;
-const rateWindow = pe.RATE_WINDOW;
-const endpointToLimit = pe.ENDPOINT_TO_LIMIT;
-const httpMethodToLimit = pe.HTTP_METHOD_TO_LIMIT;
+const rateLimit = pe.DDOS_RATE_LIMIT;
```
Comment (annyhe) [inline config.js]: out of curiosity, what does DDOS stand for?

Comment (pallavi2209) [inline config.js]: Distributed denial of service

**Pull request merged**